### PR TITLE
Rating: Added getNativeProps for events

### LINKED ISSUE #9199: Adding mouseover and mouseout events to Rating component
**Describe the feature that you would like added**
mouseover and mouseout events on Rating component
**What component or utility would this be added to**
Rating
**Have you discussed this feature with our team, and if so, who**
No
**Additional context/screenshots**
_

### PR DESCRIPTION
**Pull request by marygans**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #9199 
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Added getNativeProps so that all divProps can be passed down in ratings props.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9377)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Rating/__snapshots__/Rating.test.tsx.snap [hunk 1]
```diff
@@ -24,6 +24,8 @@ exports[`Rating Renders Rating correctly 1`] = `
         height: 32px;
       }
   id="Rating0"
+  max={5}
```
Comment (cliffkoh) [inline packages/office-ui-fabric-react/src/components/Rating/__snapshots__/Rating.test.tsx.snap]: Max and min do not look right..

File: packages/office-ui-fabric-react/src/components/__snapshots__/Rating.Basic.Example.tsx.shot [hunk 2]
```diff
@@ -28,6 +28,12 @@ exports[`Component Examples renders Rating.Basic.Example.tsx correctly 1`] = `
           height: 36px;
         }
     id="Rating0"
+    max={5}
+    min={1}
+    onBlur={[Function]}
+    onChange={[Function]}
+    onFocus={[Function]}
+    size={1}
```
Comment (cliffkoh) [inline packages/office-ui-fabric-react/src/components/__snapshots__/Rating.Basic.Example.tsx.shot]: or size for that matter.

Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/__snapshots__/Rating.Basic.Example.tsx.shot]: I'm not sure how size got there, but i see the problem with min and max, and have a local fix. The properties list hasn't been touched in 3 years, but there are a lot of things in there that shouldn't be. I'll fix that separately.

File: packages/office-ui-fabric-react/src/components/Rating/__snapshots__/Rating.test.tsx.snap (hunk 1 above)
Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/Rating/__snapshots__/Rating.test.tsx.snap]: i'll get a separate pr going to fix the getNativeProps.

File: packages/office-ui-fabric-react/src/components/__snapshots__/Rating.Basic.Example.tsx.shot (hunk 2 above)
Comment (marygans) [reply packages/office-ui-fabric-react/src/components/__snapshots__/Rating.Basic.Example.tsx.shot]: @dzearing Do I need to make any changes to PR as a result of yours?

Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/__snapshots__/Rating.Basic.Example.tsx.shot]: @marygans I just merged. So pull master into your branch. Then re-build, then update snapshots again. You should see less weird props.

Comment (cliffkoh) [reply packages/office-ui-fabric-react/src/components/__snapshots__/Rating.Basic.Example.tsx.shot]: :clap:

**Pull request merged**